server/src/Metrics.hs

```diff
@@ -25,15 +25,15 @@ data IcepeakMetrics = IcepeakMetrics
 createAndRegisterIcepeakMetrics :: IO IcepeakMetrics
 createAndRegisterIcepeakMetrics = IcepeakMetrics
   <$> registerIO (vector ("method", "status") requestCounter)
-  <*> registerIO (gauge (Info "icepeak_data_size" "Size of data file in bytes."))
+  <*> registerIO (gauge (Info "icepeak_data_size_bytes" "Size of data file in bytes."))
   <*> registerIO (gauge (Info "icepeak_journal_size_bytes"
                               "Size of journal file in bytes."))
-  <*> registerIO (counter (Info "icepeak_data_written" "Total number of bytes written so far."))
+  <*> registerIO (counter (Info "icepeak_data_written_bytes_total" "Total number of bytes written so far."))
   <*> registerIO (counter (Info "icepeak_journal_written_bytes_total"
                                 "Total number of bytes written to the journal so far."))
-  <*> registerIO (gauge (Info "icepeak_subscriber_count" "Number of websocket subscriber connections."))
+  <*> registerIO (gauge (Info "icepeak_subscribers" "Number of websocket subscriber connections."))
   where
-    requestCounter = counter (Info "icepeak_http_requests"
+    requestCounter = counter (Info "http_requests_total"
```

Comment:
why not `icepeak_http_requests_total`?

Reply:
Citing the best practices

> ...should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as namespace by client libraries. For metrics specific to an application, the prefix is usually the application name itself. Sometimes, however, metrics are more generic, like standardized metrics exported by client libraries. Examples:
>    - prometheus_notifications_total (specific to the Prometheus server)
>    - process_cpu_seconds_total (exported by many client libraries)
>    - http_request_duration_seconds (for all HTTP requests)

Especially the last example. Metrics with the same name can always be distinguished by the `node` label attached by prometheus.

Reply:
but this is specific to the icepeak server is it not?

Reply:
I think the intention is that it helps when you want to aggregate data with other services. For example a graph of all HTTP requests that arrive anywhere, even across applications. That's easy to do if the metric has the same standardized name everywhere.

You can always separate the icepeak data by filtering on the `node` label.

Reply:
But it seems they are not even consistent within their own style guide, a bit later, they have `api_http_requests_total` as an example. So I guess we can also leave the `icepeak_` prefix.
More important is, that the suffixes matches the unit and type (gauge/counter) of the metrics.